sdk/src/types/block/mana/mod.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod allotment;
+
+use alloc::{boxed::Box, vec::Vec};
+use core::ops::RangeInclusive;
+
+use derive_more::Deref;
+use hashbrown::HashSet;
+use packable::{bounded::BoundedU16, prefix::BoxedSlicePrefix, Packable};
+
+pub use self::allotment::Allotment;
+use super::{output::AccountId, Error};
+
+/// The number of bits that a given mana value can use, excluding the sign bit.
+pub const MANA_BITS: u64 = 63;
+/// Equivalent to `2^MANA_BITS - 1`
+pub const MAX_THEORETICAL_MANA: u64 = u64::MAX >> 1;
```

Comment:
I wonder about the name, why not `MAX_MANA_SUPPLY`?

Reply:
Well, that's kind of wrong since there is no supply.